exercises/practice/series/test/series_tests.erl

```diff
@@ -1,6 +1,6 @@
 %% Generated with 'testgen v0.2.0'
 %% Revision 1 of the exercises generator was used
-%% https://github.com/exercism/problem-specifications/raw/42dd0cea20498fd544b152c4e2c0a419bb7e266a/exercises/series/canonical-data.json
+%% https://github.com/exercism/problem-specifications/raw/d137db179c6a760be6aa6e38ac0110e96be0b9ff/exercises/series/canonical-data.json
```

Comment:
I'm thinking... there was been no content change to this file, but this comment was updated. This will be enough to mark the exercise as "out of date" for all students. That's a bit annoying 🤔

Reply:
Indeed. I need to change that to the last commit that touched the canonical data.

TODO: `git revert` the trivial bumps.